README.md

```diff
@@ -107,3 +107,7 @@ Now follow the [instructions on the Wiki](https://github.com/alphagov/notificati
 ## To update application dependencies
 
 `requirements.txt` is generated from the `requirements.in` in order to pin versions of all nested dependencies. If `requirements.in` has been changed, run `make freeze-requirements` to regenerate it.
+
+## Further documentation
+
+- [Guidance for deploying changes](docs/deploying.md)
```

Comment:
I think it would make more sense if ‘Guidance for deploying changes’ was in under the ‘Deploying’ H2 in the README.

Might also be worth adding a one-liner about when you might want to do the extra work in the linked document. Presumably it’s not the intention that we do all that stuff for deploying any code change or dependency update?